### ISSUE TITLE
The "Resolve block" modal has three H1 headings

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#12504]
The main heading in a modal must be an H1. As the modal content is the only perceivable content in the page, it's recommended to start the headings hierarchy from scratch with a level 1 heading.

However, the following headings in the modal must have a proper hierarchy. Currently, the modal used when a block has invalid content to "resolve" the block has three level 1 headings:

<img width="809" alt="screenshot 2018-12-01 at 16 41 07" src="https://user-images.githubusercontent.com/1682452/49330072-c50ba700-f589-11e8-85bc-1cca4872b540.png">

This modal uses the `BlockCompare` component, which in turns renders two `BlockView` components and `BlockView` has a H1 heading:

https://github.com/WordPress/gutenberg/blob/68367b4b9c869900eca07b9a9b3750ee52915c46/packages/editor/src/components/block-compare/block-view.js#L10

This should be changed to a H2 level heading.

I'd also suggest to consider some better wording:
- "Resolve Block" could be clarified a bit: rather than "resolving a block" (wondering how this can be properly translated in other languages), maybe the actual action is about resolving an error, resolving a problem or something along those lines
- "Current" doesn't tell the _what_: it's about the currently rendered markup, so maybe worth mentioning it's the Current markup or the Current block content
- same for "After Conversion": also, maybe title casing is a bit excessive in this case
- the last button "Convert to Blocks": maybe it should use the singular form "Convert to Block"?

**Comment by afercia** [comment]
Reading #9667:

Current / After conversion 

maybe they made sense when the above heading was [Compare Block Conversion](https://github.com/WordPress/gutenberg/pull/9667#issuecomment-419810587). Now that it's "Resolve Block", the two sub-headings wording is less than ideal.
